spec/features/observer_spec.rb

```diff
@@ -77,6 +77,22 @@
     expect(submit_button).to_not be_disabled
   end
 
+  it "observer can delete themselves as observer" do
```

Comment:
we already do: https://github.com/18F/C2/blob/master/spec/controllers/observations_controller_spec.rb#L34